sormas-api/src/main/resources/captions.properties

```diff
@@ -787,7 +787,8 @@ dashboardComparedTo=compared to
 dashboardComparedToPreviousPeriod=%s compared to %s
 dashboardCompletedFollowUp=Completed follow-up
 dashboardCompletedFollowUpShort=Completed F/U
-dashboardConfirmed=Confirmed
+dashboardConfirmedCase=Confirmed case
```

Comment:
The default value should still be "Confirmed" I suppose (that's what the user reads).